pyproject.toml

```diff
@@ -15,11 +15,11 @@ description = "CrateDB Python Client"
 authors = [{ name = "Crate.io", email = "office@crate.io" }]
 requires-python = ">=3.10"
 readme = "README.rst"
-license = { file = "LICENSE"}
+license = "Apache-2.0"
+license_files = ["LICENSE"]
```

Comment:
According to https://packaging.python.org/en/latest/guides/licensing-examples-and-user-scenarios/ there is no need to explicitly define `license_files` or `license_file`?
```suggestion
```